docs/reference/ilm/set-up-lifecycle-policy.asciidoc

```diff
@@ -12,6 +12,12 @@ you create the policy and add it to the index template.
 To use a policy to manage an index that doesn't roll over,
 you can specify the policy directly when you create it.
 
+IMPORTANT: When you enable {ilm} for {beats} or the {ls} {es} output plugin, 
+the necessary policies and configuration changes are applied automatically. 
+You can modify the default policies, but do not need to explicitly configure a policy or
```

Comment:
nit: This reads slightly better with an extra "you". Otherwise, I don't think the comma is needed.
But not a big deal either way. I'm fine if left as-is too.
```suggestion
You can modify the default policies, but you do not need to explicitly configure a policy or
```